reducers/navigation.js

```diff
@@ -0,0 +1,11 @@
+import { RootNavigator } from '../navigators/root.js';
+
+const splashAction = RootNavigator.router.getActionForPathAndParams('Splash');
+const initialNavState = RootNavigator.router.getStateForAction(splashAction);
+
+function navigationReducer(state = initialNavState, action) {
+    let nextState = RootNavigator.router.getStateForAction(action, state);
+    return nextState || state;
+}
+
+export default navigationReducer;
```

Comment:
Why is the first letter of the name of this reducer lowercase, but the AppReducer is camel case?